.github/workflows/go.yml

```
@@ -17,10 +17,11 @@ jobs:
 
   build:
     runs-on: ubuntu-latest
-    name: Build with Go ${{ matrix.go }} 
+    name: Build with Go ${{ matrix.go }}
     strategy:
       matrix:
-        go: [ '1.23' ]
+        go-version: [1.23.x,1.24rc1]
```

This is referenced in other places
```suggestion
        go: [1.23.x,1.24rc1]
```